paper-api/src/main/java/io/papermc/paper/statistic/Statistic.java

```diff
@@ -0,0 +1,27 @@
+package io.papermc.paper.statistic;
+
+import org.bukkit.scoreboard.Criteria;
+import org.jetbrains.annotations.ApiStatus;
+
+/**
+ * Represents an individual statistic
+ *
+ * @param <S> stat (one of {@link org.bukkit.entity.EntityType}, {@link org.bukkit.Material} or {@link CustomStatistic}).
+ */
+@ApiStatus.NonExtendable
+public interface Statistic<S> extends Criteria {
```

Comment:
Perhaps add an example here in the javadoc about how to obtain an instance of this via the StatisticType or CustomStatistic classes, since this is the first place that people who see the deprecations will be looking at.